tests/integration/test_requests.py

```diff
@@ -1,19 +1,26 @@
 # Copyright (C) 2018 SignalFx, Inc. All rights reserved.
-from random import randint
 import sys
+from random import randint
 
-from opentracing.mocktracer import MockTracer
-from opentracing.ext import tags as ext_tags
 import docker
 import pytest
-
-from requests_opentracing import SessionTracing
 import requests
+from opentracing.mocktracer import MockTracer
+from opentracing.ext import tags as ext_tags
+from packaging.version import parse as parse_version
```

Comment:
Don't believe we'd want to add an overall dependency for this test, and I think `distutils.version.StrictVersion` should work for requests fwiw.

Reply:
If you find you need the dep: https://github.com/signalfx/python-requests/blob/master/setup.py#L29

Reply:
Agree. Meant to add it as a test dep. The package is already used by setuptools if I'm not mistaken so it is not adding anything new actually. Will move to test deps.

Reply:
Updated